src/dataflow/src/sink/tail.rs

```diff
@@ -43,20 +42,18 @@ pub fn tail<G>(
                     while cursor.val_valid(&batch) {
                         let row = cursor.val(&batch);
                         cursor.map_times(&batch, |time, diff| {
+                            assert!(*diff >= 0, "negative multiplicities sinked in tail");
```

Comment:
I think you can assert strictly greater than one here as well

Reply:
idem